CONTRIBUTING.md

```diff
@@ -167,50 +204,104 @@ make license-check
-```
+---
 
-Use `git diff` afterwards to inspect the changes and ensure that they are what you expected.
+### Can you implement X in the language?
 
-OpenTofu includes generated Go stub code for the OpenTofu provider plugin protocol, which is defined using Protocol Buffers. Because the Protocol Buffers tools are not written in Go and thus cannot be automatically installed using `go get`, we follow a different process for generating these, which requires that you've already installed a suitable version of `protoc`:
+It depends. The OpenTofu language is based on the [HCL language](https://github.com/hashicorp/hcl) and the [cty typing system](https://github.com/zclconf/go-cty). Since both are available under an open source license, and we prefer to keep compatibility as much as possible, we currently don't maintain a fork of these libraries. Language features may need to be partially or fully implemented in HCL or cty and if that is the case, we can't implement them in OpenTofu without changes to the respective libraries beforehand.
 
-```
-make protobuf
-```
+---
+
+### I don't like HCL, can you replace it?
+
+HCL is baked into every corner of the OpenTofu codebase, so is here to stay. However, you can use the [JSON configuration syntax](https://opentofu.org/docs/language/syntax/json/) to write your code in JSON instead of HCL.
+
+---
+
+### Can you fix a bug in or add a feature to the Hashicorp providers, such as AWS, Azure, etc.?
+
+We currently only maintain a read-only mirror of these providers for the purposes of building binaries for the OpenTofu registry which would otherwise not be available. They are not true downstream versions that we can add patches to and OpenTofu users expect them to work the same as the upstream versions. In short: no, we cannot fix bugs or add features to these providers.
+
+---
+
+### Can you fork project X?
+
+Currently, we are at capacity for development and do not have additional capacity to take on additional projects unless necessary for the continued work on OpenTofu. While the final determination lies with the [Technical Steering Committee](#who-is-the-technical-steering-committee), the answer is likely no in almost all cases.
+
+---
+
+### Who is the core team?
+
+Core team members are full time developers sponsored by the participating companies. You can find the list of core developers in the [MAINTAINERS](MAINTAINERS) file. Their role is to triage issues, work on feature development, help plan, review, and document community contributions, support the community, and refer feature requests to the Technical Steering Committee.
+
+---
+
+### Can I become a core team member?
+
+Possibly, please look for open positions with the sponsoring companies as they hire core team members. The interview process is the same regardless of which sponsoring company you apply to. To become a core team member, you must be equally good at Go and at communication since much of our work is helping the community. Good luck!
```

Comment:
I believe this should be "Possibly. Please look...".

Reply:
It would be helpful to redirect people to a JOBS or Career page that lists the open positions from the sponsoring companies.

Reply:
Sorry, I was out of office. Unfortunately, I don't think all member companies have a dedicated careers page. :(